modules/ossm-cert-manager-update-istio-csr-revisionbased-only.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-cert-manager-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="updating-istio-csr-revision-based-only_{context}"]
+= Updating istio-csr for RevisionBased update strategy
+
+When you created your `Istio` resource, if you set the `spec.updateStrategy` field to `RevisionBased`, the `istio-csr` agent requires that you pass all revisions each time you upgrade your `RevisionBased` control plane. To do so, you must follow this order each time you upgrade:
```

Comment:
```suggestion
If you deployed your Istio resource using the revision-based update strategy, you must pass all revisions each time you update your control plane. You must perform the update in the following order:
```
See the glossary for guidance on [update vs upgrade.](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update)